erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -692,18 +692,19 @@ def update_stock_ledger(self):
 		self.make_sl_entries(sl_entries, self.amended_from and 'Yes' or 'No')
 
 	def validate_mandatory_batch_number(self):
-		# to validate the stock items batch number is present or not
+		"""validate the stock items batch number is present or not"""
 		items_without_batch_no = []
 		MANDATE_MSG = 'Batch number is mandatory for Item at : <br><b>'
```

Comment:
Question Asked : are you sure this should be < br > < b >
Ans : yes , please refer below image
![Batch_no_break](https://user-images.githubusercontent.com/80836439/123177696-62f65280-d4a3-11eb-83b8-a8333a00c3a2.png)

Reply:
```suggestion
```